packages/website/package.json

```diff
@@ -57,7 +57,6 @@
         "react-redux": "8.1.2",
         "react-router-dom": "6.16.0",
         "redux": "4.2.1",
-        "redux-devtools-extension": "2.13.9",
```

Comment:
Did you forget to add `@redux-devtools/extension`?

Reply:
Probably a mistake when merging Ill add it back